radix-engine-stores/src/hash_tree/types.rs

```diff
@@ -678,7 +679,7 @@ impl LeafKey {
 
 // SOURCE: https://github.com/aptos-labs/aptos-core/blob/1.0.4/storage/jellyfish-merkle/src/node_type/mod.rs#L48
 /// The unique key of each node.
-#[derive(Clone, Debug, Hash, Eq, PartialEq, Ord, PartialOrd)]
+#[derive(Clone, Debug, Hash, Eq, PartialEq, Ord, PartialOrd, ScryptoSbor)]
```

Comment:
Perhaps probably better to use `Sbor` rather than `ScryptoSbor` so that the hash_tree is not dependent on "scrypto" layer concepts.